components/more-less/demo/more-less.html

```diff
@@ -13,6 +13,16 @@
 
 		<d2l-demo-page page-title="d2l-more-less">
 
+						<h2>More-less with custom blur color</h2>
+
+						<d2l-demo-snippet>
+							<template>
+								<d2l-more-less blur-color="#f00">
+									<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. <a href="">Vestibulum</a> elementum venenatis arcu sit amet varius. Maecenas posuere magna arcu, quis maximus odio fringilla ac. Integer ligula lorem, faucibus sit amet cursus vel, pellentesque a justo. Aliquam urna metus, molestie at tempor eget, vestibulum a purus. Donec aliquet rutrum mi. Duis ornare congue tempor. Nullam sed massa fermentum, tincidunt leo eu, vestibulum orci. Sed ultrices est in lacus venenatis, posuere suscipit arcu scelerisque. In aliquam ipsum rhoncus, lobortis ligula ut, molestie orci. Proin scelerisque tempor posuere. Phasellus consequat, lorem quis hendrerit tempor, sem lectus sagittis nunc, in tristique dui arcu non arcu. Nunc aliquam nisi et sapien commodo lacinia. <a href="">Quisque</a> iaculis orci vel odio varius porta. Fusce tincidunt dolor enim, vitae sollicitudin purus suscipit eu.</p>
+								</d2l-more-less>
+							</template>
+						</d2l-demo-snippet>
+
```

Comment:
This is just for testing at the top of the page/scrolling up. I'll remove it before merging.